src/components/components/wizard.component.html

```diff
@@ -1,6 +1,7 @@
-<wizard-navigation-bar
-  *ngIf="navBarLocation == 'top' || navBarLocation == 'left'"
-  [ngClass]="{
+<div class="ng2-archwizard">
```

Comment:
I don't think we require a `div` wrapper here.
Can we add the class directly to the `wizard` component via the following? 
```typescript
@Component({
   host: {'class': 'ng2-archwizard'}
})
```
In general do we even require a css class for this?
I mean we have a kind of unique component name which we could use in css: `wizard`, can't we use this directly?

Reply:
Oh nice, didn't knew it was possible to add a class to a component this way. Will do this and check the result.

Will fix the conflicts and add some tests.